Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/AddBasket-AddNonprofitsViewController.m

```diff
@@ -59,8 +59,9 @@ - (IBAction)createButtonTapped:(id)sender {
 - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath {
     NonprofitListCell *cell = [tableView dequeueReusableCellWithIdentifier:@"NonprofitListCell"];
     Nonprofit *nonprofit = self.allNonprofits[indexPath.row];
-
-    //set up cell UI with nonprofit info
+    cell.nonprofitNameLabel.text = nonprofit.nonprofitName;
```

Comment:
Cell setup logic should always be moved to the cell subclass, for example you could define a `setNonprofit:` method in the `NonprofitListCell` class to update the labels and anything else on the cell. This way the cell is easier to reuse in other view controllers, and view controllers are smaller.